pallets/creator-staking/src/lib.rs

```diff
@@ -49,27 +52,37 @@ pub mod pallet {
 
         type SpacesInterface: SpacesInterface<Self::AccountId, SpaceId>;
 
+        type SpacePermissionsProvider: SpacePermissionsProvider<
+            Self::AccountId,
+            SpacePermissionsInfoOf<Self>,
+        >;
+
+        /// Specifies the amount of tokens required as a deposit for creator registration.
+        ///
+        /// This constant defines the number of tokens that a creator must lock as collateral
+        /// in order to register on the platform. The tokens act as a deposit and may be
+        /// returned under certain conditions, such as account de-registration.
         #[pallet::constant]
-        type RegistrationDeposit: Get<BalanceOf<Self>>;
+        type CreatorRegistrationDeposit: Get<BalanceOf<Self>>;
 
         /// Minimum amount can be staked to the creator.
         /// User can stake less if they already have the minimum staking amount staked to that
         /// particular creator.
         #[pallet::constant]
-        type MinimumStakingAmount: Get<BalanceOf<Self>>;
+        type MinimumStake: Get<BalanceOf<Self>>;
 
         // TODO: make it MinimumRemainingRatio
         //  (e.g. 0.1 = 10%, so that account can lock only 90% of its balance)
         /// Minimum amount that should be left on staker account after staking.
         /// Serves as a safeguard to prevent users from locking their entire free balance.
         #[pallet::constant]
-        type MinimumRemainingAmount: Get<BalanceOf<Self>>;
+        type MinimumRemainingFreeBalance: Get<BalanceOf<Self>>;
```

Comment:
Some storages in this pallet are called `Max***` but others `Minimum***`. I mean the short versions of the word vs the long version of the similar word.